dockerfiles/Caffe2Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && conda install -c anaconda cloudpickle=0.5.2 \
+      && conda install -c ezyang onnx\
+      && conda install -c conda-forge protobuf==3.4.0 \
+      && conda install -c ezyang/label/devgpu caffe2 \
+      && conda install -c ezyang onnx-caffe2 \
+      && conda install -c jjh_pytorch pytorch \
+      && conda install -c jjh_pytorch torchvision
```

Comment:
Why are you installing pytorch and torchvision in this container?

Reply:
As torch and torchvision are imported in the test file. Should I delete there lines ?

Reply:
Umm actually the python function serialization code will find PyTorch as dependencies and install them anyway, so you can leave them in. But install them as 
```
&& conda install -c pytorch pytorch torchvision \
```